homework/fibonacci/fibonacci.hpp

```diff
@@ -1,11 +1,32 @@
 #pragma once
 
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    int Fn_2{0};
+    int Fn_1{1};
```

Comment:
What are 'Fn_2' and `Fn_1` ? Variable should always represent real operation.
Like `result` `divider` `multiplier` `error_code` etc...